tests/specs/graph/jsr/FastCheck_AutoAccessors.txt

```diff
@@ -0,0 +1,92 @@
+# https://jsr.io/@scope/a/meta.json
+{"versions": { "1.0.0": {} } }
+
+# https://jsr.io/@scope/a/1.0.0_meta.json
+{ "exports": { ".": "./mod.ts" } }
+
+# https://jsr.io/@scope/a/1.0.0/mod.ts
+const symbol = Symbol();
+export class AutoAccessor {
+  /** Some comment */
+  accessor public_accessor: string;
+  private accessor private_accessor: string;
+  private accessor [symbol1]: string;
+  static static_accessor = 2;
+  private static private_static = '';
+  private static private_static_unanalyzable = unanalyzable();
+}
+
+export class PrivateAccessor {
+  accessor #private = 4;
+}
+
+# mod.ts
+import 'jsr:@scope/a'
+
+# output
+{
+  "roots": [
+    "file:///mod.ts"
+  ],
+  "modules": [
+    {
+      "kind": "esm",
+      "dependencies": [
+        {
+          "specifier": "jsr:@scope/a",
+          "code": {
+            "specifier": "jsr:@scope/a",
+            "span": {
+              "start": {
+                "line": 0,
+                "character": 7
+              },
+              "end": {
+                "line": 0,
+                "character": 21
+              }
+            }
+          }
+        }
+      ],
+      "size": 22,
+      "mediaType": "TypeScript",
+      "specifier": "file:///mod.ts"
+    },
+    {
+      "kind": "esm",
+      "size": 386,
+      "mediaType": "TypeScript",
+      "specifier": "https://jsr.io/@scope/a/1.0.0/mod.ts"
+    }
+  ],
+  "redirects": {
+    "jsr:@scope/a": "https://jsr.io/@scope/a/1.0.0/mod.ts"
+  },
+  "packages": {
+    "@scope/a": "@scope/a@1.0.0"
+  }
+}
+
+Fast check https://jsr.io/@scope/a/1.0.0/mod.ts:
+  {}
+  export class AutoAccessor {
+    /** Some comment */ declare public_accessor: string;
+    declare private private_accessor: any;
+    declare static static_accessor: number;
+    declare private static private_static: any;
```

Comment:
Wondering where the `any` is coming from. The original value is a `''` literal which could be inferred as `string` here.

Reply:
It's a private property, so I made them all `any` so that we don't have to do analysis on them or surface diagnostics.